x/committee/keeper/proposal.go

```diff
@@ -251,13 +251,32 @@ func (k Keeper) enactProposal(ctx sdk.Context, proposal types.Proposal) error {
 // CloseProposal deletes proposals and their votes, emitting an event denoting the final status of the proposal
 func (k Keeper) CloseProposal(ctx sdk.Context, proposal types.Proposal, outcome types.ProposalOutcome) {
 
+	proposalTally := types.ProposalPollingStatus{}
+	committee, found := k.GetCommittee(ctx, proposal.CommitteeID)
+	if found {
+		switch com := committee.(type) {
```

Comment:
Do we want

```
switch committee.GetType()
```

not sure how `committee.(type)` behaves for custom defined types.

Reply:
It should work just the same for custom tyes as for go types